Userland/Libraries/LibGfx/ImageFormats/ISOBMFF/Enums.h

```diff
@@ -14,11 +14,11 @@ namespace Gfx::ISOBMFF {
 
 // Define all Box types:
 #define ENUMERATE_ALL()                 \
+    ENUMERATE_ONE(FreeBox, "free")      \
     ENUMERATE_ONE(FileTypeBox, "ftyp")  \
-    ENUMERATE_ONE(MetaBox, "meta")      \
-    ENUMERATE_ONE(MovieBox, "moov")     \
     ENUMERATE_ONE(MediaDataBox, "mdat") \
-    ENUMERATE_ONE(FreeBox, "free")
+    ENUMERATE_ONE(MetaBox, "meta")      \
+    ENUMERATE_ONE(MovieBox, "moov")
```

Comment:
I was actually thinking it would be nice to group these according to their hierarchy and any ordering requirements, but I suppose that could get a bit hairy, especially if we bring indentation into the mix. Alphabetical is definitely less prone to errors, so I'm good with this, but I'm interested in your thoughts on making hierarchical groups to make it easier to find boxes related to certain parts of the different formats.

As an example, we'll have track headers `vmhd`, `smhd`, `hmhd`, `sthd`, `nmhd` all as children of `minf` (`MediaInformationBox`) in the base format, but with alphabetical ordering those will be spread out throughout this list.

Feel free to reply and resolve as you see fit.

Reply:
I thought ordering by spec order would be nice, but since the ISOBMFF spec isn't open, that's also annoying.

In the end, I thought ordering by code is easy, and it makes it easy to see if two tags have the same code, which is maybe nice. The hierarchy is visible in the callback passed to `SuperBox::read_from_stream`.

I don't have a very strong opinion about this. I'm happy if you want to change it around later :)